src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -52,7 +52,7 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->beforeNormalization()
-                ->ifTrue(function ($v) { return !isset($v['assets']) && isset($v['templating']); })
+                ->ifTrue(function ($v) { return !isset($v['assets']) && isset($v['templating']) && class_exists(Package::class); })
```

Comment:
How about moving the class check at the start of the condition ? (Yes that's kind of a micro-optimization...)

Reply:
calling a function last *is* the best it terms of µoptim to me